scripts/lib/translation-status/template.html

```diff
@@ -0,0 +1,303 @@
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+
+<head>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
+	<meta name="theme-color" content="hsl(256, 44%, 15%)" media="(prefers-color-scheme: dark)" />
+	<meta name="theme-color" content="hsl(273, 37%, 93%)" media="(prefers-color-scheme: light)" />
+
+	<title>Starlight Docs Translation Status</title>
+	<meta name="description"
+		content="Translation progress tracker for the Astro Docs site. See how much has been translated in your language and get involved!" />
+	<link rel="canonical" href="https://i18n.docs.astro.build/" />
```

Comment:
Not sure which will be the subdomain this might change.
```suggestion
	<link rel="canonical" href="https://i18n.starlight.docs.astro.build/" />
```